CMakeLists.txt

```diff
@@ -412,6 +412,10 @@ endif()
 
 option(Boost_USE_STATIC_LIBS "Use boost static libraries" ON)
 set(BOOST_COMPONENTS filesystem)
+if(MIOPEN_BUILD_DRIVER)
+    # boost core is a header only component that can't be found by find_package
+    # list(APPEND BOOST_COMPONENTS core)
```

Comment:
@atamazov all commented out, so WIP?

Reply:
@junliume The first line of the comment explains why the second line is commented out.

Reply:
@junliume All this stuff can be removed, of course. I keep this only to inform the reader.